### PR TITLE
Update Console introduction

### PR DESCRIPTION
**Pull request by randomJoe211**

### CODE REVIEW

File: docs-2.0/nebula-console.md [hunk 1]
```diff
@@ -1,14 +1,55 @@
...
+| `-P/-port` | Sets the port number of the graphd service. The default port number is 9669. |
+| `-u/-user` | Sets the username of your Nebula Graph account. Before enabling authentication, you can use any existing username. The default username is `root`. |
+| `-p/-password` | Sets the password of your Nebula Graph account. Before enabling authentication, you can use any characters as the password. |
+| `-t/-timeout`  | Sets an integer-type timeout threshold of the connection. The unit is second. The default value is 120. |
+| `-e/-eval` | Sets a string-type nGQL statement. The nGQL statement is executed once the connection succeeds. The connection stops after the result is returned. |
+| `-f/-file` | Sets the path of an nGQL file. The nGQL statements in the file are executed once the connection succeeds. The result will be returned and the connection stops then. |
+| `-enable_ssl` | Enable SSL encryption when connecting to Nebula Graph. |
```
Comment (abby-cyber) [inline docs-2.0/nebula-console.md:39]: is it better to unify the singular or plural forms of the first word? Please also check the following sentences.

File: docs-2.0/nebula-console.md [hunk 2]
```diff
@@ -145,15 +186,15 @@ Fri, 20 Aug 2021 06:36:05 UTC
 Executed 3 times, (total time spent 3681/4734 us), (average time spent 1227/1578 us)
 ```
 
-## Sleep to wait
+### Sleep to wait
```
Comment (abby-cyber) [inline docs-2.0/nebula-console.md:189]: forgive me, didn't get the meaning of sleep to wait. Is it better to say "Sleep and wait"?

File: docs-2.0/nebula-console.md (hunk 1 above)
Comment (randomJoe211) [reply docs-2.0/nebula-console.md:39]: Good catch!

File: docs-2.0/nebula-console.md (hunk 2 above)
Comment (randomJoe211) [reply docs-2.0/nebula-console.md]: Thank you for bringing this up. Hadn't noticed that there was a problem here. Modified.

**Review by abby-cyber — APPROVED** [review]

**Review by abby-cyber — APPROVED** [review]

**Review by cooper-lzy — APPROVED** [review]

**Pull request merged**